internal/cli/root.go

```diff
@@ -86,6 +86,7 @@ func Execute() {
 	// so add new commands in a place that reflect its relevance or relation with other commands:
 	rootCmd.AddCommand(loginCmd(cli))
 	rootCmd.AddCommand(tenantsCmd(cli))
+  rootCmd.AddCommand(usersCmd(cli))
```

Comment:
Seems to be malformatted. Do you use `gofmt` on save or something similar @mhsu-auth0 ?